src/morphs.jl

```diff
@@ -42,6 +42,158 @@ function morph_to(to_func::Function; do_action = :stroke)
         _morph_to(video, object, action, frame, to_func; do_action = do_action)
 end
 
+function morph_to(to_obj::Object; samples = 100)
+    return (video, object, action, frame) ->
+        _morph_to(video, object, action, frame, to_obj, samples)
+end
+
+function morph_to_fn(to_func::Function, args = []; samples = 100)
+    return (video, object, action, frame) ->
+        _morph_to_fn(video, object, action, frame, to_func, args, samples)
+end
+
+# a jpath to appear from/disappear into
+# has 1 poly of 3 points very close to each other
+# black fill 0 alpha, black stroke opaque
+# linewidth 2
+null_jpath(samples = 100) = JPath(
+    [polysample([O, O + 0.1], samples)],
+    [true],
+    [0, 0, 0, 0],
+    [0, 0, 0, 1],
+    :stroke,
+    2,
+)
+
+function _morph_to(
+    video::Video,
+    object::Object,
+    action::Action,
+    frame,
+    to_obj::Object,
+    samples,
+)
+    interp_jpaths = JPath[]
+    #nframes = length(action.frames.frames)
+    #need to handle different number of jpaths
+    #for to_obj less jpaths , we can shrink the extras down
+    #for to_obj having more jpaths , we need to create extra polys 
+    #the jpath it vanishes into has 1 poly with 3 points very close around the objects start_pos. ideally should have been 3 same points but Luxor doesnt like polys with 3 same points on top of each other,
+    l1 = length(object.jpaths)
+    l2 = length(to_obj.jpaths)
+    jpaths1 = vcat(object.jpaths, repeat([null_jpath(samples)], max(0, l2 - l1)))
+    jpaths2 = vcat(to_obj.jpaths, repeat([null_jpath(samples)], max(0, l1 - l2)))
+    #above lines should make jpaths1 and jpaths2 have the same no of jpaths
+    #
+    #resample all polys to same number of points at first frame of the action...
+    if frame == action.frames.frames[begin]
+        for obj in [object, to_obj]
+            for jpath in obj.jpaths  #kf.value is an array of jpaths
+                for i in 1:length(jpath.polys)
+                    jpath.polys[i] =
+                        polysample(jpath.polys[i], samples, closed = jpath.closed[i])
+                end
+            end
+        end
+    end
+
+    for (jpath1, jpath2) in zip(jpaths1, jpaths2)
+        push!(interp_jpaths, _morph_jpath(jpath1, jpath2, get_interpolation(action, frame)))
+    end
+    function drawfunc(args...)
+        drawjpaths(interp_jpaths)
+        global DISABLE_LUXOR_DRAW = true
+        ret = object.opts[:original_func]()
```

Comment:
It's very confusing to have this function in the middle of another function.
It doesn't seem to work like this for your first example as well. Shouldn't it be  `object.opts[:original_func](args...)`?

Reply:
Additionally the code you have in the PR description will not produce the gif there, right? Or why should the green square not be there the whole time?

Reply:
> It's very confusing to have this function in the middle of another function. It doesn't seem to work like this for your first example as well. Shouldn't it be `object.opts[:original_func](args...)`?

yes i think it should `ret = object.opts[:original_func](args...)` 

i thought it would be more clear than having an anonymous function . 
Should i change it to ...
```
object.func = (args...) -> begin
drawjpaths...
...
...
end 
```


> Additionally the code you have in the PR description will not produce the gif there, right? Or why should the green square not be there the whole time?

Yes , uploaded the wrong gif , ill fix that